### PR TITLE
Bam 45 email scheduling

### PR DESCRIPTION
**Pull request by simpsbri**
- Changes made to set the Business Last Contacted Date to the last message date for that business.
- Updated BusinessList to tag each business with a status icon based on message date.
- Updated BusinessList with filter option based on status
- Added Search box to BusinessList page
- Add email notification function for 30, 60, 90 day "lists" of businesses.
-    TODO: Update to use someone else's GMAIL
- Removed parent directory from entire project.
- Added .end to gitignore.

### CODE REVIEW

**Comment by simpsbri** [comment]
Sorry for the back and forth.....I had to make sure that your changes for the Login and Users were not lost in a conflict resolve.

**Review by michael-staples — APPROVED** [review]
lgtm

File: dist/assets/index-CYkLrT4M.js
Comment (adannavejas) [inline dist/assets/index-CYkLrT4M.js:1]: Just curious on the 'index-CYkLrT4M.js' file name format, is that intentional?

**Review by adannavejas — COMMENTED** [review]
Overall, This looks good to me!

**Pull request merged**